BurnOutSharp/ProtectionType/MGIRegistration.cs

```diff
@@ -14,8 +15,16 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (sections == null)
                 return null;
 
+            string name = pex.ProductName;
+
+            // Found in "Register.dll" in IA item "MGIPhotoSuite4.0AndPhotoVista2.02001".
+            if (name?.Equals("MGI Registration Utility", StringComparison.Ordinal) == true)
+                return $"MGI Registration {Tools.Utilities.GetInternalVersion(pex)}";
+
             // Found in "Register.dll" from "VideoWaveIII" in IA item "mgi-videowave-iii-version-3.00-mgi-software-2000".
-            var resources = pex.FindStringTableByEntry("MGI Registration");
+            var resources = pex.FindStringTableByEntry("MGI Registration")
+            // Found in "Register.dll" in IA item "MGIPhotoSuite4.0AndPhotoVista2.02001".
```

Comment:
Please split this into a separate statement instead of doing `Concat`. You're not getting a performance increase because of that.